python/vyos/ifconfig/interface.py

```diff
@@ -336,8 +366,26 @@ def __init__(self, ifname, **kargs):
         self.operational = self.OperationalClass(ifname)
         self.vrrp = VRRP(ifname)
 
+    def get_interface_within_netns(self, key):
+        if self.netns:
+            return f"ip netns exec {self.netns} {self._command_get[key]['shellcmd'].format(**self.config)}"
+        else:
+            return self._command_get[key]['shellcmd'].format(**self.config)
+
+    def set_interface_within_netns(self, key, value):
```

Comment:
Assuming the interface creation (or netns move) happens before anything else is set or moved, I'm not sure I understand the need for this function? With the exception of checking/moving/creating the interface, everything should be executed within the namespace in which the interface exists?